services/movie_session.py

```diff
@@ -0,0 +1,51 @@
+from datetime import datetime
+from django.db.models import QuerySet
+
+from db.models import MovieSession
+from services.cinema_hall import get_hall_by_id
+from services.movie import get_movie_by_id
+
+
+def create_movie_session(
+        movie_show_time: datetime,
+        movie_id: int,
+        cinema_hall_id: int
+) -> MovieSession:
+    return MovieSession.objects.create(
+        show_time=movie_show_time,
+        movie=get_movie_by_id(movie_id),
+        cinema_hall=get_hall_by_id(cinema_hall_id)
+    )
+
+
+def get_movies_sessions(session_date: str = None) -> QuerySet[MovieSession]:
+    if not session_date:
+        return MovieSession.objects.all()
+    return MovieSession.objects.filter(
+        show_time__date=session_date)
```

Comment:
In case there are no arguments on the same line with an opening parenthesis, your closing parenthesis should not have any arguments on the same line as well. Please, resolve this issue